### DISCUSSION TITLE
AdGuard is randomly resetting permissions on its own directory, which is causing all kinds of problems.

### DISCUSSION THREAD

**Discussion by jaypatrick** [AdguardTeam/AdGuardHome#7431] [Q&A]
I am running AGH on a domain. For whatever reason, recently the app started resetting permissions on its own directory, and as a result, the admin page is no longer accessible, and I can't even edit the AGH.yaml file. Even though I am a domain admin (and thus a member of the local machine's Administrator group) I am still getting locked out of editing the .yaml page so that I can get access to the admin page.

You guys are breaking all kinds of rules on Windows, stop messing with the ACLs in Windows and let the admins handle that, please. Overwriting permissions is just about the biggest mistake I've seen you guys make. Undo this change, immediately and let the users/admins take care of permissions.

**Comment by vfxraven19** [comment]
I'm getting the same issue.
